main/Card/index.tsx

```diff
@@ -34,9 +36,9 @@ const LoadingContainer = styled(Container)`
 `;
 
 const TitleContainer = styled.View<TitleContainerProps>`
-  justify-content: ${(props): string =>
-    props.hasSubTitle ? 'flex-start' : 'center'};
-  padding: 5px 10px;
+  justify-content: ${(props) => { return props.titleTextVertical ? 'flex-start' : 'center'; }};
+  padding: 3px 10px;
+  height: 46px;
   display: flex;
```

Comment:
I understand that View components have flex properties by default. Please check if this line is necessary.
(If it's for testing on the Web, you can skip it.)